src/util/Util.js

```diff
@@ -190,7 +190,7 @@ class Util extends null {
    * @returns {string}
    */
   static escapeInlineCode(text) {
-    return text.replace(/(?<=^|[^`])`(?=[^`]|$)/g, '\\`');
+    return text.replaceAll(/(?<=^|[^`])`(?=[^`]|$)/g, '\\`');
```

Comment:
Unnecessary, please revert to `replace` as RegExp with global flag will go thru all instances.
```suggestion
    return text.replace(/(?<=^|[^`])`(?=[^`]|$)/g, '\\`');
```